facia/app/controllers/FaciaController.scala

```diff
@@ -189,6 +192,9 @@ trait FaciaController
     val futureResult = futureFaciaPage.flatMap {
       case Some((faciaPage, _)) if nonHtmlEmail(request) =>
         successful(Cached(CacheTime.RecentlyUpdated)(renderEmail(faciaPage)))
+      case Some((faciaPage: PressedPage, _)) if FaciaPicker.getTier(faciaPage, path)(request) == RemoteRender =>
+        val pageType = PageType(faciaPage, request, context)
+        remoteRenderer.getFront(ws, faciaPage, pageType)(request)
```

Comment:
Remote renderer already gives us a Future<Cached> which is why we're not using the existing Cached below

Reply:
Good catch! Thank you!

Reply:
Ignore me if this has already been addressed but I think @alinaboghiu had some thoughts on this?